pkgs/by-name/ap/apftool-rs/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "apftool-rs";
+  version = "unstable-2024-01-05";
```

```suggestion
  version = "0-unstable-2024-01-05";
```
`version` must always start with a digit.